modules/installation/pages/install-proxy-uyuni.adoc

```diff
@@ -0,0 +1,48 @@
+[[install-proxy-uyuni]]
+= Install Uyuni Proxy with openSUSE
+
+{productname} Proxy can be installed on openSUSE.
+
+
+
+== Install Uyuni Proxy on openSUSE Leap 15.1
+
+.Procedure: Installing openSUSE Leap 15.1 with Uyuni
+
+. As the base system, install openSUSE Leap 15.1 with all available package updates applied.
+
+. Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].
+
+. Add the repository for installing the {productname} Proxy software as [systemitem]``root``:
++
+
+// variable assignment to avoid overlong lines
+// https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
++
+----
+repo=repositories/systemsmanagement:/
+repo=${repo}Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
+zypper ar https://download.opensuse.org/$repo uyuni-proxy-stable
+----
+
+. Refresh metadata from the repositories as [systemitem]``root``:
++
+
+----
+zypper ref
+----
+
+. Install the pattern for the {productname} Server as [systemitem]``root``:
++
+
+----
+zypper in patterns-uyuni_proxy
+----
+
+. Reboot.
+
+* For more information about the stable version of {productname}, see https://www.uyuni-project.org/pages/stable-version.html.
```

Comment:
Remove bullets.

Reply:
But several one-sentence paras in a row are even worse, IMO.  It's a list, and that's why I think a formal list is fine. If we'd connect the two items and have one para, it would also look bad.  I do not know what would be the best.

Reply:
fair enough :)